lib/ecstatic/show-dir/index.js

```diff
@@ -24,6 +24,15 @@ module.exports = function (opts, stat) {
 
   return function middleware (req, res, next) {
 
+    if (req.url.endsWith('?css')) {
+      res.writeHead(200, {
+        'Content-Type': 'text/css',
+        'Cache-Control': 'public, max-age=31536000'
```

Comment:
I feel like ideally cache headers should behave the same as in the rest of the app.

I get that maybe directory listings don't properly do this now, but that would also be a bug.